var/spack/repos/builtin/packages/py-cogent/package.py

```diff
@@ -31,7 +31,19 @@ class PyCogent(PythonPackage):
     homepage = "http://pycogent.org"
     url      = "https://pypi.io/packages/source/c/cogent/cogent-1.9.tar.gz"
 
-    version('1.9', '7d9f28cd17664c1cd18c568fc53060d6')
+    version('1.9', sha256='57d8c58e0273ffe4f2b907874f9b49dadfd0600f5507b7666369f4e44d56ce14')
+    version('1.5.3', url="https://github.com/pycogent/pycogent/archive/1.5.3-release.tar.gz", 
+        sha256='4e19325cd1951382dc71582eb49f44c5a19eb128e3540e29dc28e080091e49cd')
 
-    depends_on('py-setuptools',   type='build')
-    depends_on('py-numpy',        type=('build', 'run'))
+    variant('matplotlib', default=False, description="graphs related to codon usage")
+    variant('mpi4py', default=False, description='MPI required for parallel computation.')
```

Comment:
I would call this variant `mpi` and default to True. This is what the `setup.py` calls it:
```python
"mpi": ["mpi4py"],                                          
```